src/schema/objects/columns.yaml

```diff
@@ -0,0 +1,507 @@
+---
+HED:
+  name: HED
+  description: |
+    Hierarchical Event Descriptor (HED) Tag.
+    See [Appendix III](/99-appendices/03-hed.html) for details.
+  type: string
+abbreviation:
+  name: abbreviation
+  description: |
+    The unique label abbreviation
+  type: string
+acq_time:
+  name: acq_time
+  description: |
+    Acquisition time refers to when the first data point in each run was acquired.
+    Furthermore, if this header is provided, the acquisition times of all files that
+    belong to a recording MUST be identical.
+
+    Datetime should be expressed as described in [Units](/02-common-principles.html#units).
+
+    For anonymization purposes all dates within one subject should be shifted by a randomly
+    chosen (but consistent across all recordings) number of days.
+    This way relative timing would be preserved, but chances of identifying a person based
+    on the date and time of their scan would be decreased.
+    Dates that are shifted for anonymization purposes SHOULD be set to the year 1925 or earlier
+    to clearly distinguish them from unmodified data.
+    Shifting dates is RECOMMENDED, but not required.
+  type: string
+  format: datetime
+age:
+  name: age
+  description: |
+    Numeric value in years (float or integer value).
+  type: number
+  unit: year
+cardiac:
+  name: cardiac
+  description: |
+    continuous pulse measurement
+  type: number
+color:
+  name: color
+  description: |
+    Hexadecimal. Label color for visualization.
+  type: string
+  unit: hexadecimal
+derived_from:
+  name: derived_from
+  description: |
+    `sample-<label>` key/value pair from which a sample is derived,
+    for example a slice of tissue (`sample-02`) derived from a block of tissue (`sample-01`).
+  type: string
+  pattern: ^sample-[0-9a-zA-Z]+$
+description:
+  name: description
+  description: |
+    Brief free-text description of the channel, or other information of interest.
+  type: string
+dimension:
+  name: dimension
+  description: |
+    Size of the group (grid/strip/probe) that this electrode belongs to.
+    Must be of form `[AxB]` with the smallest dimension first (for example, `[1x8]`).
+  type: string
+duration:
+  name: duration
+  description: |
+    Duration of the event (measured from onset) in seconds.
+    Must always be either zero or positive.
+    A "duration" value of zero implies that the delta function or event is so
+    short as to be effectively modeled as an impulse.
```

Comment:
why did you cut out:

>  (or "n/a" if unavailable)

? :)

Reply:
I honestly have no idea 😕. I'll add it back now though!